sphinx/themes/basic/static/searchtools.js

```diff
@@ -97,6 +97,14 @@ const _displayItem = (item, searchTerms, highlightTerms) => {
     fetch(requestUrl)
       .then((responseData) => responseData.text())
       .then((data) => {
+
+        // exclude results that don't contain exact phrases if we are searching for them
+        if (data) {
+          const lowercaseData = data.toLowerCase();
+          const mismatch = (s) => lowercaseData.search(s) === -1;
```

Comment:
This can be expressed more concisely with includes:

```suggestion
          const mismatch = (s) => !lowercaseData.includes(s);
```